components/README.md

```diff
@@ -0,0 +1,15 @@
+# Libraries for Compose Multiplatform
```

Comment:
Can we follow general KKM project structure?

Reply:
Yes

Reply:
I will change structure

Reply:
Almost done.
Leave JS for not as it was.